Sluggo/View Controllers/RootViewController.swift

```diff
@@ -56,4 +58,14 @@ class RootViewController: UIViewController {
     @IBSegueAction func createTicket(_ coder: NSCoder) -> TicketListController? {
         return TicketListController(coder: coder, identity: identity)
     }
+    
+    @IBAction func receivedGesture() {
+        print("swiped right")
```

Comment:
is this placeholder functionality? is this not intended to expand/collapse the sidebar?

Reply:
I didn't because I was under the assumption that you were eventually going to handle this. I can take care of it.

Reply:
I will handle on my branch